How To - Guides/How to add clothes/How to add clothes.txt

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------
+------------------- How to add clothes-------------------
+---------------------------------------------------------
```

Comment:
I'm not sure this is really a good idea to document, since the codebase can just change right? I think the way to do this is to break it into component parts

Reply:
ah I see what you mean! Yeah probably best to break it into components - good point